### DISCUSSION TITLE
Guidance on Script development workflow

### DISCUSSION THREAD

**Discussion by rapatel0** [SilentVoid13/Templater#324] [Help]
Hey Guys, 

I was wondering how people go about developing and running scripts. I've been toying around with some JS scripts, but I havent' found a great way to test and debug scripts, other than running them on test notes and hoping for the best. 

Is there a better way to test code via the console? Something to access the right scope and variables?

**Comment by bbendick** [comment]
I was hoping to find something similar. Even the tiny thing I'm trying to do it's a real pain to create a new note every time to test it out. Were you able to find a good workflow on your own @rapatel0 ?
